src/docs/style/logo.mdx

```diff
@@ -0,0 +1,267 @@
+-   Our [Figma Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) also includes a component called "App Logo Template"
+
+## Glyph
+
+A glyph is a simple shape representing an object, action, or idea. Your glyph must be simple, recognizable, and scalable to ensure that your app can be easily identified on multiple devices with different resolutions.
+
+We recommend you select your base glyph from our [icon library](/style/icon-library).
+
+<ImageGrid
+    images={[Glyph1, Glyph2]}
+    captionsUnderImages={[
+        `AVOID: Avoid the use of mnemonic logos or logotypes when possible. They are often not meaningful for first-time users, and can be hard to see in small logo sizes.`,
+        `DO: Use a graphical way to represent an application or product.`,
+    ]}
+/>
+
+Do not use the Eaton logo as a glyph — your app should not be presented as the "Eaton" app. It is not unusual for a customer to have multiple Eaton products and applications installed on their devices, and if every one of those apps used the Eaton logo it becomes very difficult for a user to tell, at a glance, which application they are looking for.
+
+<ImageGrid images={[Glyph3]} captionsUnderImages={[`DONT: Do not use the Eaton logo as glyphs.`]} />
+
+You should also verify that your product logo is distinguished enough from other existing Eaton product logos.
+
+Glyphs are usually aligned to the keyline (visible in the template) to avoid bleeding beyond the safe zone.
+
+<ImageGrid images={[GlyphKeyline]} captionsUnderImages={[`Align glyphs to the icon template keyline.`]} regularWidth />
+
+Once the base glyph is decided, you can be creative and add variations to the glyph to make it look more designed.
+
+<ImageGrid
+    images={[PolishGlyph1, PolishGlyph2]}
+    captionsUnderImages={[
+        `Smart Energy Manager modified the glyph shape to accommodate for a bolt glyph inside.`,
+        `Predict Pulse modified the glyph size, opacity, and image gradient.`,
+    ]}
+/>
+
+## Background
+
+Most Eaton app logos use [the primary blue](/style/color?category=ui&name=blue&weight=500) as the base background color. Some also select the background color from one of the [branding colors](/style/color#branding-colors) to stay consistent with colors in their product marketing.
+
+<ImageGrid
+    images={[Background1]}
+    captionsUnderImages={[`Use the primary blue or one of the branding colors as a foundation for the background.`]}
+    regularWidth
+/>
+
+<ImageGrid
+    images={[Background2, Background3]}
+    captionsUnderImages={[
+        `DONT: Do not use a photo in the logo.`,
+        `DONT: Do not use a screenshot or app UI in the logo.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[BackgroundGradient1, BackgroundGradient2, BackgroundGradient3]}
+    captionsUnderImages={[
+        `DO: Most Eaton apps choose to add a subtle corner linear gradient to their app logo background.`,
```

Comment:
```suggestion
        `DO: Most Eaton apps choose to add a subtle linear gradient to corner of their app logo background.`,
```

Reply:
not technically true... that gradient spans the whole background

Reply:
'corner linear gradient' sounds weird to me. How about 'a diagonal gradient'

Reply:
that is exactly what I ended up doing :D